sdk/keyvault/azure-keyvault-secrets/tests/test_secrets_client.py

```diff
@@ -115,11 +115,9 @@ def _update_secret(secret):
         updated = _update_secret(created)
 
         # delete secret
-        deleted = client.delete_secret(updated.name)
+        deleted = client.begin_delete_secret(updated.name).result()
         self.assertIsNotNone(deleted)
 
-        self._poll_until_exception(functools.partial(client.get_secret, updated.name), ResourceNotFoundError)
```

Comment:
Are these helpers unused now?

Reply:
we're still using this to ensure that secrets are properly purged.